### PR TITLE
Homework: Added shadows and styles for TrainViewController buttons

### PR DESCRIPTION
**Pull request by Bryant-24**

### CODE REVIEW

File: Math-Trainer/Math-Trainer/TrainViewController.swift [hunk 1]
```diff
@@ -7,12 +7,23 @@
 
 import UIKit
 
-final class TrainViewController: UIViewController {
+final class TrainViewController: ViewController {
+
```
Comment (AntonPastrevich) [inline Math-Trainer/Math-Trainer/TrainViewController.swift]: extra line

File: Math-Trainer/Math-Trainer/TrainViewController.swift [hunk 2]
```diff
@@ -8,11 +8,32 @@
...
         didSet {
             print(type)
         }
     }
+
+    // MARK: - Life cycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureButtons(buttonsAnswerCollection)
+        configureButtons([buttonBack])
```
Comment (AntonPastrevich) [inline Math-Trainer/Math-Trainer/TrainViewController.swift]: u can call this method once 
`configureButtons(buttonsAnswerCollection + [buttonBack])`

**Review by AntonPastrevich — APPROVED** [review]

File: Math-Trainer/Math-Trainer/TrainViewController.swift (hunk 1 above)
Comment (Bryant-24) [reply Math-Trainer/Math-Trainer/TrainViewController.swift]: Removed, thanks!

File: Math-Trainer/Math-Trainer/TrainViewController.swift (hunk 2 above)
Comment (Bryant-24) [reply Math-Trainer/Math-Trainer/TrainViewController.swift]: Yep, done, thanks!

**Pull request merged**